content/_policy/rules-of-use._es.md

```diff
@@ -28,7 +28,7 @@ El servicio Login.gov protege su cuenta mediante la aplicación de fuertes medid
 2. La divulgación sea exigida por ley; o
 3. La divulgación sea necesaria para investigar un fraude relacionado con su cuenta.
 
-Algunas aplicaciones gubernamentales que utilizan Login.gov requieren que usted verifique su identidad. Esto significa que debe demostrar que usted es quien dice ser. Esa prueba ayuda a garantizar que sólo las personas adecuadas tengan acceso a información sensible. Sólo se le pedirá que verifique su identidad la primera vez que acceda a determinadas aplicaciones gubernamentales. Después de verificar su identidad con Login.gov para una aplicación gubernamental, normalmente no necesitará volver a hacerlo para otras aplicaciones gubernamentales. Sin embargo, cuando acceda a IRS.gov por primera vez, siempre tendrá que verificar su identidad aunque ya lo haya hecho para otra aplicación. Al proporcionar al servicio Login.gov la información necesaria para verificar su cuenta, usted le autoriza a utilizar dicha información para llevar a cabo las actividades necesarias para garantizar su identidad, incluido el intercambio de información con determinados terceros, especialmente los servicios de comprobación de identidad. El servicio Login.gov exige a dichas partes que mantengan su información de forma confidencial y que no la utilicen para otros fines, tal y como se detalla en nuestra [política de privacidad.]({% link _policy/privacy._es.md %})
+Algunas aplicaciones gubernamentales que utilizan Login.gov requieren que usted verifique su identidad. Esto significa que debe demostrar que usted es quien dice ser. Esa prueba ayuda a garantizar que sólo las personas adecuadas tengan acceso a información sensible. Sólo se le pedirá que verifique su identidad la primera vez que acceda a determinadas aplicaciones gubernamentales. Después de verificar su identidad con Login.gov para una aplicación gubernamental, normalmente no necesitará volver a hacerlo para otras aplicaciones gubernamentales. Sin embargo, cuando acceda a IRS.gov por primera vez, siempre tendrá que verificar su identidad aunque ya lo haya hecho para otra aplicación. Al proporcionar al servicio Login.gov la información necesaria para verificar su cuenta, usted le autoriza a utilizar dicha información para llevar a cabo las actividades necesarias para garantizar su identidad, incluido el intercambio de información con determinados terceros, especialmente los servicios de comprobación de identidad. El servicio Login.gov exige a dichas partes que mantengan su información de forma confidencial y que no la utilicen para otros fines, tal y como se detalla en nuestra [política de privacidad.](/es/policy/)
```

Comment:
In English and French versions, we keep the period outside the link. Should it be the same for Spanish?

Also, same note as previously with using the link helper.

```suggestion
Algunas aplicaciones gubernamentales que utilizan Login.gov requieren que usted verifique su identidad. Esto significa que debe demostrar que usted es quien dice ser. Esa prueba ayuda a garantizar que sólo las personas adecuadas tengan acceso a información sensible. Sólo se le pedirá que verifique su identidad la primera vez que acceda a determinadas aplicaciones gubernamentales. Después de verificar su identidad con Login.gov para una aplicación gubernamental, normalmente no necesitará volver a hacerlo para otras aplicaciones gubernamentales. Sin embargo, cuando acceda a IRS.gov por primera vez, siempre tendrá que verificar su identidad aunque ya lo haya hecho para otra aplicación. Al proporcionar al servicio Login.gov la información necesaria para verificar su cuenta, usted le autoriza a utilizar dicha información para llevar a cabo las actividades necesarias para garantizar su identidad, incluido el intercambio de información con determinados terceros, especialmente los servicios de comprobación de identidad. El servicio Login.gov exige a dichas partes que mantengan su información de forma confidencial y que no la utilicen para otros fines, tal y como se detalla en nuestra [política de privacidad]({% link _policy/privacy._es.md %}).
```